rundk/README.md

```diff
@@ -0,0 +1,46 @@
+# rundk
+
+`rundk` is a tool to run a test command from the test image, by using it
+developers can reproduce the test flow as run in the CI environment.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/) must be installed
+- If you run this tool on MAC, `/tmp` directory must be added to Docker's File
+  Sharing settings to allow Docker accessing the temporary files created for
+  mount, see the [comment](https://github.com/docker/docker.github.io/issues/4709#issuecomment-639596451).
+
+## Installation
+
+`rundk` can be installed and upgraded by running:
+
+```shell
+go get knative.dev/test-infra/rundk
+```
+
+## Usage
+
+```shell
+Usage of rundk:
+  -test-image string
+        The image we use to run the test flow. (default "gcr.io/knative-tests/test-infra/prow-tests:stable")
+  -mounts string
+        A list of extra folders or files separated by comma that need to be mounted to run the test flow.
+  -mandatory-env-vars string
```

Comment:
These two options don't mention that the env vars get promoted to the image

Reply:
Updated the README here.

Reply:
I don't see this change?